quick-start/examples/n6.min.js

```diff
@@ -0,0 +1,24 @@
+import { Scene, Circle, Rectangle } from "https://unpkg.com/pencil.js/dist/pencil.esm.js";
+
+const scene = new Scene(undefined, {
+    fill: "#45a8ff",
+});
+const { width, height } = scene;
+// Initialise the target position
```

Comment:
```suggestion
// Initialize the target position
```
Usage of a British spelling in an otherwise American English text.